ArticleTemplates/assets/scss/pillar/_colour.scss

```diff
@@ -341,8 +341,76 @@
                 background: $p-feature-headline;
                 color: $p-inverted;
             }
+
+            &.personality-quiz {
+                .question__wrapper {
+                    border-top: 2px solid color(brightness-7);
+                }
+
+                .question__text {
+                    &::before {
+                        color:  color(brightness-7);
+                    }
+                }
+
+                .quiz-scores,
+                .quiz-results {
+                    border-color: color(brightness-7);
+                    border-top: 2px solid color(brightness-7);
+                }
+            }
+        }
+    }
+
+    @media (prefers-color-scheme: dark) {
```

Comment:
Do you know that we can use .ios & .android classes in here as well? to indicate if the style changes is for ios or android?

Reply:
the prefers-color-scheme is only for iOS and the dark-mode-on is only for android. adding the .ios and .android classes would have no benefit here

Reply:
yeaa fair enough 👍